src/app.js

```diff
@@ -0,0 +1,35 @@
+import "./style.scss";
+import Logo from './assets/img/Logo.svg'
+import Photo from './assets/img/Photo.svg'
+
+let portfolio_img = [];
+
+function importAll(r) {
```

Comment:
Rename the function. The name should make it easier for a reader of your code  to understand what is happening here.
The name itself is now not clear, the parameter `r` is also not understandable.  Shorter syntax is not always better.